pkg/provisioner/ironic/ironic.go

```diff
@@ -1707,7 +1707,7 @@ func (p *ironicProvisioner) Deprovision(force bool) (result provisioner.Result,
 // Delete removes the host from the provisioning system. It may be
 // called multiple times, and should return true for its dirty flag
 // until the deprovisioning operation is completed.
-func (p *ironicProvisioner) Delete() (result provisioner.Result, err error) {
+func (p *ironicProvisioner) Delete(force bool) (result provisioner.Result, err error) {
```

Comment:
Given that the `force` parameter is not used anymore, we probably don't need to add it.